docs/primehub-app-tutorial-label-studio.md

```diff
@@ -0,0 +1,72 @@
+---
+id: primehub-app-tutorial-label-studio
+title: Label Dataset by Label Studio
+description: Label Dataset by Label Studio
+---
+<div class="label-sect">
+  <div class="ee-only tooltip">Enterprise
+    <span class="tooltiptext">Applicable to Enterprise Edition</span>
+  </div>
+  <div class="ce-only tooltip">Community
+    <span class="tooltiptext">Applicable to Community Edition</span>
+  </div>
+</div>
+<br>
+
+This tutorial covers the basic flow to help you get started with Label Studio in PrimeHub.
+
+## Install Label Studio
+First, you need to install it in the `Apps` tab. Please check the [Overview](primehub-app) section to learn how to install an App. 
+
+In the installing process, you can change the environment variables.
+
+`DEFAULT_USERNAME` and `DEFAULT_PASSWORD` are the login account information. You can change them and use them to log into Label Studio after installed.
+
+If you don't know the meaning of other environment variables, you can use the default values or check the [Label Studio Official Doc](https://github.com/heartexlabs/label-studio/blob/master/docs/source/guide/start.md) or the tooltip beside the environment variable for more details.
+
+## Label Studio UI
+PrimeHub shows the app's state in the `Apps` tab. You can open the Label Studio UI by clicking `Open` after the state becomes `Ready`.
+
+![](assets/app_tutorial_labelstudio_login_page.png)
+
+It will open a new window and show the Label Studio UI. You can find your login information by clicking `Manage` in the `Apps` tab and then clicking the eyes icon. The `$(PRIMEHUB_GROUP)` is the group name.
+
+![](assets/app_tutorial_labelstudio_login_info.png)
+
+## Label Dataset
+### What we need?
+
+- The dataset in PrimeHub you want to label (we use `/datasets/dog-demo` in this tutorial)
```

Comment:
Suggest to add hyper link at dataset to https://docs.primehub.io/docs/next/guide_manual/admin-dataset